src/main/scala/org/allenai/pipeline/s3/S3Pipeline.scala

```diff
@@ -11,6 +11,7 @@ import java.net.URI
 
 trait S3Pipeline extends Pipeline {
   def credentials: S3Credentials = S3Config.environmentCredentials()
+  implicit val s3Cache: S3Cache = DefaultS3Cache
```

Comment:
To turn off persistent caching, override this val with `LocalS3Cache(None)` in the top-level pipeline class